### ISSUE TITLE
Searching for a single character is broken

### ISSUE THREAD

**Issue by asklar** [microsoft/terminal#7607]
# Environment

```none
Windows build number: 19041
Windows Terminal version (if applicable):

Any other software?
```

# Steps to reproduce

in cmd running on conhost, Ctrl+F and search for a **single character**

# Expected behavior

search works and finds the next occurrence of the character

# Actual behavior

nothing happens

**Comment by DHowett** [comment]
I believe this was fixed after VB shipped. I can't reproduce it on 20000+, but I can reproduce it with the 19041 conhost.

Use terminal.

**Comment by asklar** [comment]
I'd love to but I can't use terminal if it breaks my muscle memory with functionality such as keyboard selection etc. not working, sorry.

**Comment by DHowett** [comment]
It looks like this was broken in !1768108 (2018!) and fixed incidentally in #3590. Resolving fixed -- it'll come out inbox with whatever comes after 19041!
